res_company_code/views/assets.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    @author Dave Lasley <dave@laslabs.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <template id="assets_backend" inherit_id="web.assets_backend">
+        <xpath expr=".">
+            <script type="application/javascript"
+                    src="/res_company_code/static/src/js/res_company_code.js"
```

Comment:
No such file :)

Reply:
indeed. the whole file assets.xml is dead code. removed.